common/src/main/java/net/draycia/carbon/common/command/commands/LeaveCommand.java

```diff
@@ -90,7 +90,7 @@ public void init() {
                     this.carbonMessages.channelNotFound(sender);
                     return;
                 }
-                final ChannelPermissionResult permitted = channel.speechPermitted(sender);
+                final ChannelPermissionResult permitted = channel.joinPermitted(sender);
```

Comment:
feels a little weird to check if they can join a channel to leave it, could just change the method name though